### PR TITLE
Redirect flow

### PR DESCRIPTION
**Pull request by andreicioban**
### What changed?
The subscribe flow changes from `in place` (sending a `jsonp` request) to `redirect` as in `stirioficiale.ro`. 
Also changed the styling to match the one on `stirioficiale.ro`.

### CODE REVIEW

File: src/components/subscribe/subscribe-form.js [hunk 1]
```diff
@@ -1,47 +1,20 @@
...
-      onSubmit({
-        email: inputValue
-      });
-    }
-  };
-
+export const SubscribeForm = ({ compact }) => {
   const SubscribeButton = () => (
-    <Button onClick={handleSubmit} disabled={loading || success}>
+    <Button onClick={() => void 0} inputType="submit">
```
Comment (moonflare) [inline src/components/subscribe/subscribe-form.js]: We can remove `onClick`, it's been removed from the component
```suggestion
    <Button inputType="submit">
```

**Comment by moonflare** [comment]
@andreicioban Can you also please rebase?

**Comment by aniri** [comment]
@andreicioban the build has failed. can you please check? there might be an issue with the dependencies in the package-lock file

**Comment by andreicioban** [comment]
@aniri it builds fine on my machine. Sometimes Now doesn't/report properly.

**Comment by aniri** [comment]
Hey @andreicioban . It's not just now. I also get this error on my computer :( The last time this showed up, I deleted node_modules and package-lock and regenerated them. Can you please do this? Also, the 0.1.15 version of the components was not released yet, so there's no need to increase it yet :)

**Pull request merged**